content/en/get-started/overview.md

```diff
@@ -2,10 +2,16 @@
 title: How to Get Started using Armory CD-as-a-Service
 linkTitle: Overview
 description: >
-  Armory CD-as-a-Service...
+  Methods of getting started with Armory CD-as-a-Service
 weight: 1
 categories: ["Get Started", "Concepts"]
 tags: ["Quickstart", "Deployment"]
 ---
 
```

Comment:
This is a great start. I think this page could use cardpanes/cards with different user journeys...

Also what paths to take if you don't have a K8s cluster -> use the UI sandbox route, use kind to install locally...

more thinking needed for future iterations